functions/vcs.zsh

```diff
@@ -110,7 +110,18 @@ function +vi-hg-bookmarks() {
 
 function +vi-vcs-detect-changes() {
   if [[ "${hook_com[vcs]}" == "git" ]]; then
-    vcs_visual_identifier='VCS_GIT_ICON'
+
+    local remote=`git ls-remote --get-url`
+    if [[ "$remote" =~ "github" ]] then
```

Comment:
Yes, `git ls-remote --get-url` gives me something like `git@github.com:bhilburn/powerlevel9k.git` therefore I wanted to do more like a sub-string search by only looking for: github, bitbucket or gitlab words.

If you think this can be improved, please let me know how, I would gladly change it.